files/get-ecr-uri.sh

```diff
@@ -63,10 +63,55 @@ else
     il-central-1)
       acct="066635153087"
       ;;
-    *)
+    # This sections includes all commercial non-opt-in regions, which use
+    # the same account for ECR pause container images, but still have in-region
+    # registries.
+    ap-northeast-1 | \
+      ap-northeast-2 | \
+      ap-northeast-3 | \
+      ap-south-1 | \
+      ap-southeast-1 | \
+      ap-southeast-2 | \
+      ca-central-1 | \
+      eu-central-1 | \
+      eu-north-1 | \
+      eu-west-1 | \
+      eu-west-2 | \
+      eu-west-3 | \
+      sa-east-1 | \
+      us-east-1 | \
+      us-east-2 | \
+      us-west-1 | \
+      us-west-2)
       acct="602401143452"
       ;;
-  esac
+    *)
+      # If the region is not mapped to an account, let's try to choose another region
+      # in that partition.
+      case $aws_domain in
```

Comment:
wait a minute -- this value is a TLD, not a partition name. E.g. for `aws-cn`, it's `amazonaws.com.cn`, right?

Reply:
Doh! Good catch. I'm just going to use the region name since this is just a fallback anyway.